handlers/register_test.go

```diff
@@ -56,6 +56,8 @@ func TestUnitRegister(t *testing.T) {
 		So(router.GetRoute("disqualified-officer-delta-validate"), ShouldNotBeNil)
 		So(router.GetRoute("company-delta"), ShouldNotBeNil)
 		So(router.GetRoute("company-delta-validate"), ShouldNotBeNil)
+		So(router.GetRoute("psc-delta"), ShouldNotBeNil)
```

Comment:
This file is failing in concourse:
* /tmp/build/9780d8b7/go/src/github.com/companieshouse/chs-delta-api/handlers/register_test.go 
  Line 60:
  Expected '<nil>' to NOT be nil (but it was)!
https://ci.platform.aws.chdev.org/teams/common/pipelines/chs-delta-api/jobs/analyse-pull-request/builds/88